cloud_AWS/terraform/module/examples/multiple-aws-accounts-multiple-vpc-setup/aws_onboarder.py

```diff
@@ -23,8 +23,12 @@ class AwsProfile:
     secret_key: str
 
 
-TerraformAction = Callable[[Terraform, str], bool]  # terraform plan/apply/destroy for specified region
-RequestedProfileNames = Optional[List[str]]  # list of profile names matching profiles in ~/.aws/credentials
```

Comment:
Hmm why does this line break, it is under 120 chars long?

Reply:
Looks like the linter doing the line-breaking is configured to 100 chars per line. This should be fixed. We want 120 chars per line.